.opencode/command/openai.md

```
@@ -0,0 +1,5 @@
+---
+description: Interact with Linear for issue/project management
+agent: NormalOpenAI
+---
+$ARGUMENTS
```

markdownlint-cli2 (0.22.0)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)